superset-frontend/src/dashboard/components/nativeFilters/ColumnSelect.tsx

```diff
@@ -65,7 +65,12 @@ export function ColumnSelect({
       endpoint: `/api/v1/dataset/${datasetId}`,
     }).then(
       ({ json: { result } }) => {
-        return result.columns.map((col: any) => col.column_name);
+        return result.columns
+          .map((col: any) => col.column_name)
+          .sort(
+            (a: ColumnSelectValue['label'], b: ColumnSelectValue['label']) =>
```

Comment:
Not super strong opinion, but this might be clearer if we just use the `string` type here.

Reply:
I was hesitating for these types here, so I like the idea for simplify that to "string", thanks.